client/src/modules/chapters/pages/chaptersPage.tsx

```diff
@@ -14,7 +15,12 @@ export const ChaptersPage: NextPage = () => {
 
   return (
     <Stack mt={10} mb={5} display={'block'}>
-      <Heading marginBlock={'1em'}>Chapters: </Heading>
+      <Flex alignContent={'center'}>
+        <Heading marginBlock={'1em'}>Chapters: </Heading>
+        <LinkButton href="/dashboard/chapters" colorScheme={'blue'}>
+          Chapter Dashboard
+        </LinkButton>
```

Comment:
It's a little misaligned
![image](https://user-images.githubusercontent.com/15801806/193256106-5a45740e-95d1-4841-a5c0-0e4a6f7875ac.png)

Also, can we hide this for logged out users?

Reply:
Do we need to hide them? we can disable them if they aren't logged in

at least this way people won't think, we are hiding stuff 🤔. but this maybe me overthinking it

Reply:
I have added user import so hiding them is easy, what do we want to hide or disable the buttons?